NeuroRuler/utils/parser.py

```diff
@@ -18,13 +18,17 @@ def parse_cli() -> None:
     """Parse CLI (non-GUI) args and set settings in `cli_settings.py`.
 
     :return: None"""
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(
+        prog="NeuroRuler",
+        description="A program that calculates head circumference from MRI data (`.nii`, `.nii.gz`, `.nrrd`)."
+    )
+
     parser.add_argument("-d", "--debug", help="print debug info", action="store_true")
     parser.add_argument("-r", "--raw", help="print just the \"raw\" circumference", action="store_true")
-    parser.add_argument("-x", "--x", type=int, help="x transformation (in degrees)")
-    parser.add_argument("-y", "--y", type=int, help="y transformation (in degrees)")
-    parser.add_argument("-z", "--z", type=int, help="z transformation (in degrees)")
-    parser.add_argument("-s", "--slice", type=int, help="slice (0-indexed)")
+    parser.add_argument("-x", "--x", type=int, help="x rotation (in degrees)")
+    parser.add_argument("-y", "--y", type=int, help="y rotation (in degrees)")
+    parser.add_argument("-z", "--z", type=int, help="z rotation (in degrees)")
+    parser.add_argument("-s", "--slice", type=int, help="slice (cervical, 0-indexed)")
```

Comment:
Should this say "axial"?

Reply:
Martin said a "C slice", I assume that means cervical? (could be very wrong)

Reply:
I don't recall hearing "C slice"

![image](https://user-images.githubusercontent.com/69875698/234656142-f48f2e2e-e0ce-4cc0-8afb-e41503272fcd.png)

(a) sagittal (b) axial (c) coronal

Reply:
I heard "C slice". The view is axial yes, but I'm not sure the slice is.